lib/please_cli/please_cli/decision_task.py

```diff
@@ -443,16 +472,65 @@ def cmd(ctx,
         if not project_exists_in_cache:
             build_projects.append(project)
 
-    projects_to_deploy = []
+    click.echo(' => Checking github for project revisions')
+
+    project_revisions = {p: github_commit for p in PROJECTS}
+    project_url_cache = dict()
+    for project in sorted(PROJECTS):
+        # TODO: how should we handle outside projects?
+        #       skip project defined outside for now
+        if project in please_cli.config.OUTSIDE_PROJECTS:
```

Comment:
Maybe let the outside project specify a full taskcluster route name in their own config ?